src/gsi/balmod.f90

```diff
@@ -419,12 +408,30 @@ subroutine prebal_reg(cwcoveqqcov)
        endif
     enddo j_loop
     
-    agvk=zero
```

Comment:
I think some of the zeroing out got improperly consolidated.  It looks like the first *vk=zero block is to initialize the variables before the loops.  The second block for zeroing out the *vk coefficients in the IF block is to turn the balance off.  I don't think that these can be combined.

Reply:
I don't understand your concern.  There is nothing different from zeroing out the arrays for initializing the variable or for turning off the balance.  There is really no reason to initialize the variables to zero if they are then going to be overwritten.  I don't understand why you think they cannot be combined.

Reply:
Ah sorry, I missed the "else" and thought it was an "endif" for some reason and thought they were still being overwritten.  Ignore.